EasyPost/Constants.cs

```diff
@@ -90,5 +90,17 @@ public static class ErrorMessages
             internal const string ApiErrorDetailsParsingError = "RESPONSE.PARSE_ERROR"; // not for public consumption
             public const string CouldNotPassClient = "Could not pass client to {0}.";
         }
+
+        public static class CarrierAccountTypes
+        {
+            public const string FedExAccountType = "FedexAccount";
+            public const string UpsAccountType = "UpsAccount";
+
+            internal static List<string> CustomWorkflowCarrierAccountTypes => new List<string>
+            {
+                FedExAccountType,
+                UpsAccountType
+            };
+        }
```

Comment:
This is so much code where it's not necessary. KISS.

Let's have it be as simple as this psuedo code:

`CARRIER_TYPES_WITH_CUSTOM_WORKFLOWS = ['FedexAccount', 'UpsAccount']`

We need to be writing the minimal amount of code possible to get tests to pass/behavior met. The more code we have, the more liability we have. Let's design for what we need today (always) and refactor for what we need tomorrow when tomorrow comes because sometimes tomorrow doesn't even come. If ever there comes a day where there is another carrier with a custom workflow (unlikely), we'd just append a new string to the simple list.

TLDR: Let's just use a flat constant instead of a whole class for this.

Reply:
I agree. My one concern is, if someone were to send `type="FedExAccount"` (not FedEx vs Fedex), it wouldn't be captured by the list, and it would attempt to hit the normal endpoint with the data (granted, the data is bad anyway; assuming the API is case-sensitive for account type strings)

Reply:
Valid, though I thiiink the API would fail because we validate the type there so it may be a non-issue